pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -788,3 +788,16 @@ def test_mi_columns_loc_list_label_order():
         columns=MultiIndex.from_tuples([("B", 1), ("B", 2), ("A", 1), ("A", 2)]),
     )
     tm.assert_frame_equal(result, expected)
+
+
+def test_mi_partial_indexing_list_raises():
+    # GH 13501
+    frame = DataFrame(
+        np.arange(12).reshape((4, 3)),
+        index=[["a", "a", "b", "b"], [1, 2, 1, 2]],
+        columns=[["Ohio", "Ohio", "Colorado"], ["Green", "Red", "Green"]],
+    )
+    frame.index.names = ["key1", "key2"]
+    frame.columns.names = ["state", "color"]
+    with pytest.raises(KeyError, match="\\[2\\] not in index"):
+        frame.loc[["b", 2], "Colorado"]
```

Comment:
this is bc the user passed a list instead of a tuple?

Reply:
Correct, as a tuple I think this should correctly select

```
In [2]: frame
Out[2]:
state      Ohio     Colorado
color     Green Red    Green
key1 key2
a    1        0   1        2
     2        3   4        5
b    1        6   7        8
     2        9  10       11

In [3]:  frame.loc[('b', 2), 'Colorado']
Out[3]:
color
Green    11
Name: (b, 2), dtype: int64
```